docs/extend/extend-apiml/websocket.md

```diff
@@ -13,41 +22,66 @@ The service can define what WebSocket services are exposed using Eureka metadata
                     routes:
                         ws_v1:
                             gatewayUrl: "ws/v1"
-                            serviceUrl: /discoverableclient/ws
+                            serviceUrl: /${serviceId}/ws
 
-These metadata make it possible for requests from `wss://gatewayHost:port/serviceId/ws/v1/path` to map to `ws://serviceHost:port/discoverableclient/ws/path`,
-where:
+These metadata make it possible for requests from `wss://gatewayHost:gatewayPort/${serviceId}/ws/v1/path` to map to `wss://serviceHost:servicePort/${serviceId}/ws/v1/path`, where:
 
 * **`serviceId`**  
-  is the service ID of the service
+  is the service ID of the service.
   
 * **`path`**  
   is the remaining path segment in the URL.
 
-## Security and Authentication
+## Architecture
+
+In WebSocket routing, the API ML Gateway acts as both a WebSocket server for the client requesting this connection, and as a WebSocket client.
+
+Client (i.e. web browser) <-> Gateway (WebSocket Server) - Gateway (WebSocket Client) <-> Service's WebSocket Server
```

Comment:
Perhaps we should add a line to say what this arrangement is. Something like, "The following schema describes the arrangement between client-side and server-side components where the Gateway serves as both the Websocket server as well as the Websocket client."... or something like that.